tools/system_libs.py

```diff
@@ -41,7 +41,7 @@ def read_symbols(path, exclude=None):
       symbols = filter(lambda symbol: symbol not in exclude, symbols)
     return set(symbols)
 
-  default_opts = ['-Werror']
+  default_opts = ['-Werror', '-Wno-error=absolute-value']
```

Comment:
How about

1. Adding that to the list I see exists in `build_libc`, so it just applies to musl (and not other system libs), and
2. Adding it if `shared.Settings.WASM_BACKEND`, as we don't need it in asm.js

? lgtm with that.